aya/src/maps/lpm_trie.rs

```diff
@@ -99,6 +119,38 @@ impl<K: Pod> Clone for Key<K> {
 // A Pod impl is required as Key struct is a key for a map.
 unsafe impl<K: Pod> Pod for Key<K> {}
 
+impl<K: Pod, V: Pod> LpmTrie<MapData, K, V> {
+    /// Create an LPM Trie from userspace.
+    pub fn create(name: &str, max_entries: u32) -> Result<LpmTrie<MapData, K, V>, MapError> {
```

Comment:
I'd rather not do this. At a minimum we should add a flags argument too.

I do think we want to make it possible to create maps, but we should do it for
all maps for consistency, and we need to figure out the rest of the API first,
for instance https://github.com/aya-rs/aya/pull/540